include/APIClient/Router.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <string>
+#include <map>
+
+namespace APIClient {
+
+namespace Router {
+
+    enum class Route : uint8_t {
+        getAll,
+        getVulkanInfoWithId,
+        put
+    };
+
+    static constexpr const char* baseAPIUri = LUGBENCH_API_URI;
+    static constexpr const char* apiVersion = LUGBENCH_API_VERSION;
+    static constexpr const char* apiVersionUri = "api/";
+
+    std::string getUrlString(Route route, std::string id);
```

Comment:
So, is it really okay to always take an id even if two of the three URI don't take an id?